src/test/java/com/uid2/shared/secure/AttestationTokenTest.java

```diff
@@ -67,6 +67,20 @@ public void testAttestationTokenNew() {
         Assertions.assertTrue(reconstructToken.getExpiresAt().isAfter(expiryLowerBound));
         Assertions.assertTrue(reconstructToken.getExpiresAt().isBefore(expiryUpperBound));
     }
+    @Test
+    public void testAttestationTokenNewDefaultLifetime() {
```

Comment:
I think it's better if the test name describes what actual capability is being tested, "new default lifetime" is vague and won't be new any more soon.

Reply:
Updated the name of the test